app-text/xpdf/xpdf-3.04.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="Open source Portable Document Format (PDF) viewer"
+DESCRIPTION="The Xpdf project also includes a PDF text extractor, PDF-to-PostScript \
+converter, and various other utilities.
+
+Xpdf runs under the X Window System on UNIX and OS/2.  The non-X \
+components (pdftops, pdftotext, etc.) also run on Windows and Mac OSX \
+systems and should run on pretty much any system with a decent C++ \
+compiler.  Xpdf will run on 32-bit and 64-bit machines."
+HOMEPAGE="http://www.foolabs.com/xpdf"
+COPYRIGHT="1996-2014 Glyph & Cog, LLC."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://mirror.physik-pool.tu-berlin.de/tex-archive/support/xpdf/xpdf-$portVersion.tar.gz"
+CHECKSUM_SHA256="11390c74733abcb262aaca4db68710f13ffffd42bfe2a0861a5dfc912b2977e5"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
> When this recipe builds on x86_gcc2, you can drop SECONDARY_ARCHITECTURES

@miqlas: If you are sure that the recipe builds on x86_gcc2 then you can (and probably should) also remove all occurrences of **`$secondaryArchSuffix`** (and **`${secondaryArchSuffix}`**).

Reminder: Of course, if this recipe had at least one **`lib:lib*`** (or **`devel:lib*`**) in its PROVIDES or PROVIDES_devel then it would have been OK to keep the SECONDARY_ARCHITECTURES definition and all occurrences of secondaryArchSuffix.